src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -193,12 +192,15 @@ $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
                     // TODO: request should be delayed if there is a viewerRequestVolume
                     return func(result, layer).then((content) => {
                         tile.add(content);
+                        if (content.RTC) {
+                            content.RTC.add(content);
```

Comment:
Why don't you do this directly in the loaders?
Every user of .b3dm or .pnts will have to write the exact same code, so in my opinion it should be moved to the loader.